File: 프로그래머스/unrated/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.java
import java.util.*;
class Solution {
    public int[] solution(String my_string) {
        my_string = my_string.replaceAll("[a-z]", ""); 
        
        String[] num = my_string.split("");
        int[]answer = new int[num.length];
        
        for (int i = 0; i < num.length; i++) {
        	answer[i] = Integer.parseInt(num[i]);
		}
        Arrays.sort(answer);
        
    	return answer;
    }
}

File: 프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.js
function solution(my_string) {
    var answer = [];
    for (let i = 0; i < my_string.length; i++) {
        if (/\d/.test(my_string[i])) {  
            answer.push(parseInt(my_string[i]));  
        }
    }
    answer.sort((a, b) => a - b);  
    return answer;
}
